plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogConfig.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog.nessie;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import java.net.URI;
+
+public class IcebergNessieCatalogConfig
+{
+    private String defaultReferenceName = "main";
+    private String defaultWarehouseDir;
+    private URI serverUri;
+
+    @NotNull
+    public String getDefaultReferenceName()
+    {
+        return defaultReferenceName;
+    }
+
+    @Config("iceberg.nessie-catalog.ref")
+    @ConfigDescription("The default Nessie reference to work on")
+    public IcebergNessieCatalogConfig setDefaultReferenceName(String defaultReferenceName)
+    {
+        this.defaultReferenceName = defaultReferenceName;
+        return this;
+    }
+
+    @NotNull
+    public URI getServerUri()
+    {
+        return serverUri;
+    }
+
+    @Config("iceberg.nessie-catalog.uri")
+    @ConfigDescription("The URI to connect to the Nessie server")
+    public IcebergNessieCatalogConfig setServerUri(String serverUri)
```

Comment:
```suggestion
    public IcebergNessieCatalogConfig setServerUri(URI serverUri)
```

Reply:
Even the `IcebergRestCatalogConfig` has `String` for `setBaseUri` method.  
Do we need to change this?

Reply:
Yes.

Reply:
ok. I hope airlift will convert the string from config file to the URI as this argument is annotated with `@Config`